.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,86 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+title: "[BUG] <title>"
+labels: ['bug, untriaged']
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Related component
+      description: Choose a specific OpenSearch component your bug belongs to.  If you are unsure which to select or if the component is not present, select "Other".
+      multiple: false
+      options:
+        - Build
+        - Clients
+        - Cluster Manager
+        - Extensions
+        - Indexing
+        - Indexing:Performance
+        - Indexing:Replication
+        - Libraries
+        - Other
+        - Plugins
+        - Search
```

Comment:
Nit: may be it will be better if we move the Search label under other search label, to give the chance to user to specify sub-area (e.g. Search:Performance) instead of choosing the generic Search area. Same comment for Indexing

Reply:
updated